### DISCUSSION TITLE
Automation/CRON/Generate recurring events

### DISCUSSION THREAD

**Discussion by krseguros** [YetiForceCompany/YetiForceCRM#17216] [How to]
Hello,
What can be happening here? everything was fine and suddenly I notice that something is wrong with the flow Generate recurring events "without completion"

How can i fix this?
![image](https://github.com/YetiForceCompany/YetiForceCRM/assets/96985476/40dae2c2-9c54-49b3-aaa0-e3db9c611cda)

**Comment by mariuszkrzaczkowski** [comment]
requires verification of logs
